### PR TITLE
doc: update README.md to reflect current usage

### PR DESCRIPTION
**Pull request by mhdawson**
Now that N-API has landed in core update to reflect
how we are using the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 across different versions and flavors of Node.js without recompilations.
 
 It is introduced by this Node enhancemnet proposal:
 [005-ABI-Stable-Module-API.md](https://github.com/nodejs/node-eps/blob/master/005-ABI-Stable-Module-API.md).
 
-This is project is in experimental stage at the moment. During the latest [VM Summit](https://github.com/nodejs/vm/issues/4), we [discussed the progress](https://github.com/nodejs/abi-stable-node/blob/doc/VM%20Summit.pdf) of this project. The summit participants agreed to consider a PR for this feature as experimental in the upcoming Node.js version 8.0. You can checkout the [Notes from Node.js VM Summit](https://blogs.windows.com/msedgedev/2017/03/13/notes-from-nodejs-vm-summit/) for more details.
+N-API is now part of core as an experimental feature. Documentation is available here:
+[https://nodejs.org/docs/latest/api/n-api.html]().
 
 **Branches**
```
Comment (aruneshchandra) [inline README.md:15]: Maybe we should add the video some place. 
https://www.youtube.com/watch?v=nmXhJ88nZsk

Comment (aruneshchandra) [inline README.md:15]: Also, we should update the "How to get involved section" from the text in the blog.

**Comment by mhdawson** [comment]
I'm going to land this and then submit an additional update as we want it to be more up to date for when we give the presentation at NI.

**Comment by mhdawson** [comment]
Landed as 693dfb4540b2ac173cf3100b90ef7bf12a5c7069

**Pull request closed without merging**